COMPILING.md

```diff
@@ -0,0 +1,161 @@
+Jamulus can be compiled for Linux, Windows and macOS as follows. Alernatively, you may wish to use one of the contributed [installation scripts](https://github.com/jamulussoftware/installscripts).
+
+---
+
+
+## Download sources
+
+For .tar.gz [use this link](https://github.com/jamulussoftware/jamulus/archive/latest.tar.gz) to download the latest release
+
+For .zip [use this link](https://github.com/jamulussoftware/jamulus/archive/master.zip)
+
+## Linux
+
+#### Install dependencies
+
+On Ubuntu-based distributions 18.04+, Debian 9+ or 10 and Raspberry Pi Raspbian Buster release or later:
+
+* Build-essential
+* Qt5-qmake
+* Qtdeclarative5-dev
+* Qt5-default
+* Qttools5-dev-tools
+* Libjack-jackd2-dev
+
+#### On Fedora
+
+* Qt5-qtdeclarative-devel
+* Jack-audio-connection-kit-dbus
+* libQt5Concurrent5
+* Jack-audio-connection-kit-devel
+
+#### For all desktop distributions
+
+[QjackCtl](https://qjackctl.sourceforge.io/) is optional, but recommended to configure JACK.
+
+
+### Standard desktop build
+
+~~~
+qmake Jamulus.pro
+make clean
+make
+sudo make install
+~~~
+
+This puts the Jamulus binary into `/usr/local/bin`.
+
+#### Notes
+
+* The "make clean" command is essential to remove the automatically generated Qt files which are present in the .tar.gz file and may not match the Qt version you are using.
+
+* To use an external shared OPUS library instead of the built-in one use qmake `"CONFIG+=opus_shared_lib" Jamulus.pro`.
+
+* To use this file configure the software with `qmake "CONFIG+=noupcasename" Jamulus.pro` to make sure the output target name of this software is **j**amulus instead of **J**amulus.
+
+* Users of Raspberry Pi: You may want to compile the client on another machine and run the binary on the Raspberry Pi. In which case the only libraries you need to run it are those for a [headless server](Server-Linux#running-a-headless-server) build, but _with_ the JACK sound packages. In particular, have a look at the footnote for the headless build.
+
+* As of version 3.5.3, Jamulus is no longer compatible with Qt4.
+
+
+### “Headless” server build
+
+Note that you don’t need to install the JACK package(s) for a headless build. If you plan to run headless on Gentoo, or are compiling under Ubuntu for use on another Ubuntu machine, see the footnote.
+
+Also, although not strictly necessary, we recommend using the headless flag to speed up the build process. Gentoo users may also be able to avoid installing some dependencies as a consequence. For more information [see footnote](#footnote).
+
+Compile the sources to ignore the JACK sound library:
+
+~~~
+qmake "CONFIG+=nosound headless" Jamulus.pro
+make clean
+Make
```

Comment:
```suggestion
make
```

Reply:
Just committed it since I think it's a small thing.